projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -1119,12 +1252,15 @@ function settings_select( $name, $values, $extra_text = '' ) {
 		}
 		echo '</select>';
 		if ( ! empty( $extra_text ) ) {
-			echo '<p class="description">' . $extra_text . '</p>';
+			echo '<p class="description">' . esc_textarea( $extra_text ) . '</p>';
```

Comment:
Same as above, `esc_textarea()` is wrong here. Should probably be `esc_html()` or some sort of `wp_kses()` (maybe `wp_kses_allowed_html()`?).

Reply:
`wp_kses_post` here too possibly?